scala/scala.bzl

```diff
@@ -162,9 +162,31 @@ def _compile(ctx, cjars, dep_srcjars, buildijar):
     all_srcjars = set(srcjars + list(dep_srcjars))
     # look for any plugins:
     plugins = _collect_plugin_paths(ctx.attr.plugins)
+
+    if (hasattr(ctx.attr, 'enable_dependency_analyzer')
+        and ctx.attr.enable_dependency_analyzer
```

Comment:
i think this (the ability to turn off dependency analyzer) doesn't have a test, right?
even if this is covered by existing tests (which I'm not sure) there's definitely no test which encodes the feature, right? WDYT about having the next pair also add that test?